docs/guides/faq.md

```diff
@@ -0,0 +1,120 @@
+the video.
+
+## Q: I see an error thrown that mentions `vdata12345`. What is that?
+This error is thrown when an element that is associated with a component is removed
+from the DOM but the event handlers associated with the element are not removed. This
+is almost always due to event listeners not being disposed when dispose is called on
+a component.
+
+## Q: What media formats does video.js support?
+This depends on the browsers support, video.js will work off of that. If Flash is
+avialable Flash videos can also be played but only if the Flash tech is included with
+video.js.
+
+## Q: How can I hide the links to my video/subtitles/audio/tracks?
+This is outside of the scope of video.js and there are things that can be done to make
+it harder, but it will never be completely possible.
+
+## Q: I think I found a bug with video.js or I want to add a feature. What should I do?
+### if you think that you can fix the issue or add the feature
+Submit a pull request to the [video.js repo](https://github.com/videojs/video.js/pulls).
+Make sure to follow the [contributing guide](/CONTRIBUTING.md#contributing-code) and
+the [pull request template](/.github/PULL_REQUEST_TEMPLATE.md).
+
+### If you don't think you can fix the issue or add the feature
+Open an [issue on the video.js repo](https://github.com/videojs/video.js/issues). Make
+sure that you follow the [issue template](/.github/ISSUE_TEMPLATE.md) and the
+[contributing guide](/CONTRIBUTING.md#filing-issues) so that we can better assist you
+with your issue.
+
+## Q: What is a [reduced test case](https://css-tricks.com/reduced-test-cases/)?
+A [reduced test case](https://css-tricks.com/reduced-test-cases/) is an example of the
+problem that you are facing in isolation. Think of it as example page that reproduces
+the issue in the least amount of possible code. We have a [starter example](http://jsbin.com/axedog/edit?html,output)
+for reduced test cases.
+
+## Q: How do a make a plugin for video.js?
+See the [plugin guide](/docs/guides/plugins.md) for information on making a plugin for video.js.
+
+## Q: How do I add a button to video.js?
+See the [button guide](/docs/guides/button.md) for information on adding a button to video.js.
+
+## Q: Where can I find a list of video.js plugins?
+A list of plugins is avialable on [videojs.com](http://videojs.com/plugins).
+
+## Q: How can I get my plugin listend on the website?
+Add the 'videojs-plugin' [keyword to your array in package.json](https://docs.npmjs.com/files/package.json#keywords)
+and publish your package to npm.
+
+## Q: Where can I find a list of video.js skins?
+See the [video.js github wiki](https://github.com/videojs/video.js/wiki/Skins).
+
+## Q: Does video.js work as an audio only player?
+Yes! It can be used to play audio only files in a `<video>` or `<audio>` tag. The
+difference being that the `<audio>` tag will not have a blank display area and the `<video>`
+tag will.
```

Comment:
While Flash is still around, worth noting that audio only doesn't work in Flash.